berkeley-mobile/Fitness/GymDetailViewController.swift

```diff
@@ -38,6 +40,15 @@ class GymDetailViewController: SearchDrawerViewController {
         middleCutoffPosition = (openTimesCard?.frame.maxY ?? overviewCard.frame.maxY) + scrollingStackView.yOffset + 8
     }
 
+    /// Opens `gym.website` in Safari. Called as a result of tapping on `moreButton`.
+    @objc private func moreButtonClicked(sender: UIButton) {
+        guard let url = gym.website else { return }
+        presentAlertLinkUrl(title: "Are you sure you want to open Safari?",
+                            message: "Berkeley Mobile wants to open this resource's website",
```

Comment:
nitpick: do we want to say "resource's" here if we're opening a gym website? could also be the parks and stuff that backend added, we should probably figure out terminology. top of the card says "Fitness Centers" which isn't quite right either.
maybe "Fitness Location" or "Fitness Resource"?

Reply:
Renamed the top of the card to say "Fitness Locations", and updated the alert message!